src/lib/componentloader.js

```diff
@@ -7,29 +7,29 @@ ComponentLoader.prototype = {
   loadComponentsData: function () {
     var xhr = new window.XMLHttpRequest();
     // @todo Remove the sync call and use a callback
-    xhr.open('GET', 'https://raw.githubusercontent.com/aframevr/aframe-components/master/components.json', false);
+    xhr.open('GET', 'https://aframe.io/aframe-registry/build/' + AFRAME.version + '.json');
```

Comment:
you have to restrict this to the major version:

- if it's 0.3.x, use 0.3.0
- if it's 4.x.x, use 4.0.0